src/services/StreamService.ts

```diff
@@ -28,7 +28,10 @@ class StreamService {
     // Create from a test result with multiple test types, multiple test result with one test type each
     const records: any[] = event.Records.filter((record: DynamoDBRecord) => {
       // Retrieve "INSERT" events
-      return record.eventName === "INSERT" || record.eventName === "MODIFY";
+      return (
+        record.eventName === "INSERT" ||
+        (record.eventName === "MODIFY" && process.env.MODIFY_EVENTS)
```

Comment:
I think for readability it might be better for this env variable to be called `IGNORE_MODIFY_EVENTS`

Reply:
Done, renamed to PROCESS_MODIFY_EVENTS